### DISCUSSION TITLE
iPadOS: Crash when drawing menu screen on versions higher than 2023.617

### DISCUSSION THREAD

**Discussion by LordXerus** [ppy/osu#24637] [Q&A] [answered]
Username: LordXerus
IPadOS:16.1.1

The game can open and update the database, however after the intro sequence the game will seize up and crash. This is happening on all versions currently newer than 617 (currently latest is 815). I’m hoping someone with access to TestFlight logs can tell me what is going on. Thanks in advance.

**Comment by peppy** [comment]
It may be that your renderer is set to something non-standard.

If you don't mind losing data, can you try deleting and reinstalling the app?

**Reply by LordXerus** [reply, marked as answer]
That worked, after a little more testing, it seems to be caused by the OpenGL renderer. 
After reinstalling, switching to OpenGL and performing some actions such as logging in or downloading beat maps will crash the game.

**Reply by peppy** [reply]
We're aware of this one. Please leave the renderer setting alone for now!

**Reply by LordXerus** [reply]
Yeah OpenGL renderer is just not quite functional on my end. Using it makes solo song select carousel crash and multi lobby selection crash. I’m not sure what caused the crash on load, however.

**Reply by peppy** [reply]
Already tracked at https://github.com/ppy/osu/issues/24651!
